global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "10.0.100-preview.7.25380.108",
+    "version": "9.0.305",
```

Comment:
```suggestion
    "version": "10.0.100-preview.7.25380.108",
```

Reply:
Thanks for doing this - I made a mental note to switch it back and forgot.